cli/upgrade.rs

```diff
@@ -56,8 +57,20 @@ pub async fn upgrade_command(
   dry_run: bool,
   force: bool,
   version: Option<String>,
+  ca_file: Option<String>,
 ) -> Result<(), ErrBox> {
-  let client = Client::builder().redirect(Policy::none()).build()?;
+  let mut client_builder = Client::builder().redirect(Policy::none());
+
+  // If we have been provided a CA Certificate, add it into the HTTP client
+  if let Some(ca_file) = ca_file {
+    let mut buf = Vec::new();
+    File::open(ca_file)?.read_to_end(&mut buf)?;
```

Comment:
`std::fs::read(ca_file)` is shorter

Reply:
I copied and pasted it from another part of the code base, want me to shorten it down?

Reply:
If you don't mind - thank you

Reply:
All done :)